config/plugins/interactive_environments/askomics/templates/askomics.mako

```diff
@@ -0,0 +1,50 @@
+<%namespace name="ie" file="ie.mako" />
+<%
+import random
+import string
+# Sets ID and sets up a lot of other variables
+ie_request.load_deploy_config()
+
+# Get a random API key
+askomics_api_key = ''.join([random.choice(string.ascii_letters + string.digits) for n in xrange(20)])
+
+# Get ids of selected datasets
+additional_ids = trans.request.params.get('additional_dataset_ids', None)
+if not additional_ids:
+    additional_ids = str(trans.security.encode_id( hda.id ) )
+else:
+    additional_ids += "," + trans.security.encode_id( hda.id )
```

Comment:
In case of None: Is `additional_ids` then `None, 45`

Reply:
Thanks for your feedback @bgruening, but I don't anderstand what you mean here.

Reply:
When `additional_ids = trans.request.params.get('additional_dataset_ids', None)` is None, what is the value of additional_ids after the if condition?

Reply:
When the IE is lauched by clicking the vizualise button of a dataset, `trans.request.params.get('additional_dataset_ids', None)` is `None`. After the condition, `additional_ids` is the selected dataset id.